### PR TITLE
Closes #74

### LINKED ISSUE #74: Load starter images to disk
Use the ImageManager class to load Item images in XCAssets to the filesystem on app startup.

Retrieve images from disk using ImageManager for use in collection view, detail view.

### PR DESCRIPTION
**Pull request by csuttner**
- Moves initial loading of data from MockDatabase into Core Data into AppDelegate
- Handles creation of an images directory (sub of documents) if not already present from previous run
- Implements methods necessary to load files from xcassets into this directory
- Adds loading of xcassets images for plants in the loadInitData() method

### CODE REVIEW

File: PhotoPoints/Model/Repository.swift [hunk 1]
```diff
@@ -85,6 +100,29 @@ class Repository {
...
+        return nil
+    }
+    
+    func getImageFromFilesystem(item: Item) -> UIImage? {
+        
+        if let firstImage = item.images?.allObjects.first as? Image {
+            let fileName = firstImage.filename
+            print(ImageManager.getPath(for: fileName))
+            return ImageManager.getImage(from: fileName)
+        }
```
Comment (gomezfox) [inline PhotoPoints/Model/Repository.swift]: May need to cast `URL?` to `String` for line 119 to behave (this is presumably a for logging)

**Review by Grant-Buchanan — APPROVED** [review]

**Review by gomezfox — CHANGES_REQUESTED** [review]
Need to cast `URL?` to `String` for print function

File: PhotoPoints/Model/Repository.swift (hunk 1 above)
Comment (csuttner) [reply PhotoPoints/Model/Repository.swift]: lets talk about this in the AM

Comment (csuttner) [reply PhotoPoints/Model/Repository.swift]: It is working as is by the way

**Pull request merged**